### PR TITLE
Add VerifyingKey::to_edwards

### PR DESCRIPTION
**Pull request by randombit**
See #623

### CODE REVIEW

File: ed25519-dalek/src/verifying.rs [hunk 1]
```diff
@@ -505,6 +505,11 @@ impl VerifyingKey {
     pub fn to_montgomery(&self) -> MontgomeryPoint {
         self.point.to_montgomery()
     }
+
+    /// Return this verifying key in Edwards form.
+    pub fn to_edwards(&self) -> EdwardsPoint {
+        self.point
+    }
```
Comment (tarcieri) [inline ed25519-dalek/src/verifying.rs:512]: Perhaps add a `From` impl too?

Comment (randombit) [reply ed25519-dalek/src/verifying.rs:512]: There is already a `impl From<EdwardsPoint> for VerifyingKey` - is your suggestion here to add the cooresponding `impl From<VerifyingKey> for EdwardsPoint`?

**Comment by tarcieri** [comment]
Hmm, an unrelated failure due to new nightly lints for a debugging trait which needs to get annotated with `#[allow(dead_code)]`

File: ed25519-dalek/src/verifying.rs (hunk 1 above)
Comment (randombit) [reply ed25519-dalek/src/verifying.rs:512]: Done

**Comment by tarcieri** [comment]
@randombit #625 is merged. If you rebase it should fix the test failure

**Comment by rozbb** [comment]
You can merge `main` now to fix the CI errors

**Comment by randombit** [comment]
Rebased with #625

**Comment by rozbb** [comment]
Thank you!

**Pull request merged**